build.gradle

```diff
@@ -25,3 +26,14 @@ dependencies {
 tasks.named('test') {
 	useJUnitPlatform()
 }
+
+jacoco {
```

Comment:
Nice! I've never heard of this library but its definitely good to see code coverage 👍